grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -496,21 +497,25 @@ def _GetDescribedSchema(
 
   def _GetParameters(
       self,
-      path_params: List[FieldDescriptor],
+      required_path_params: List[FieldDescriptor],
+      optional_path_params: List[FieldDescriptor],
```

Comment:
I don't think you care about this being a list, `Iterable` should be enough. Try to be as liberal in parameter type as possible.

Reply:
Noted. Thank you for pointing this out, I made the change!